client/src/components/LandingNavbar/useStyles.ts

```diff
@@ -2,7 +2,19 @@ import { makeStyles, Theme } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
-    flexGrow: 1,
+    padding: '0 2rem',
+    '@media (max-width: 900px)': {
+      paddingLeft: 0,
+      paddingRight: 0,
+    },
+    '@media (max-width: 600px)': {
+      position: 'relative',
+    },
+  },
+  menuItem: {
+    '&.MuiListItem-button:hover': {
+      backgroundColor: 'initial',
+    },
   },
```

Comment:
You can probably just move this over to the button itself as there is no guarantee the class name will be the same in future versions of MUI.

Reply:
Will do that